Default to Debian alternative x-terminal-emulator
--- stumpwm.orig/bindings.lisp
+++ stumpwm/bindings.lisp
@@ -79,8 +79,8 @@
   *escape-key* '*root-map*)
 
 (fill-keymap *root-map*
-  (kbd "c")   "exec xterm"
-  (kbd "C-c") "exec xterm"
+  (kbd "c")   "exec x-terminal-emulator"
+  (kbd "C-c") "exec x-terminal-emulator"
   (kbd "e")   "emacs"
   (kbd "C-e") "emacs"
   (kbd "b")   "banish"
--- stumpwm.orig/sample-stumpwmrc.lisp
+++ stumpwm/sample-stumpwmrc.lisp
@@ -19,7 +19,7 @@
 ;; Browse somewhere
 (define-key *root-map* (kbd "b") "colon1 exec firefox http://www.")
 ;; Ssh somewhere
-(define-key *root-map* (kbd "C-s") "colon1 exec xterm -e ssh ")
+(define-key *root-map* (kbd "C-s") "colon1 exec x-terminal-emulator -e ssh ")
 ;; Lock screen
 (define-key *root-map* (kbd "C-l") "exec xlock")
 
